index.html

```diff
@@ -1469,6 +1475,11 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
     The <dfn>NFCReader.onreading</dfn> is an EventHandler which is called to notify
     that new reading is available.
   </p>
+  <p>
+    The <dfn>NFCReader.url</dfn>, <dfn>NFCReader.recordType</dfn>,
+    <dfn>NFCReader.mediaType</dfn> and <dfn>NFCReader.accept</dfn> correlates to
+    the members of <a>NFCReaderOptions</a>.
```

Comment:
Nit: s/correlates the/have the values of the corresponding/ (when correlates is used as a transitive verb, the subject is the person doing the analysis)